src/processIssueComment.js

```diff
@@ -42,18 +42,24 @@ async function processAddContributor({
         originalSha: optionsConfig.getOriginalSha(),
     }
 
-    const pullRequestURL = await repository.createPullRequestFromFiles({
-        title: `docs: add ${who} as a contributor`,
-        body: `Adds @${who} as a contributor for ${contributions.join(
-            ', ',
-        )}.\n\nThis was requested by ${commentReply.replyingToWho()} [in this comment](${commentReply.replyingToWhere()})`,
-        filesByPath: filesByPathToUpdate,
-        branchName: `all-contributors/add-${who}`,
-    })
+    if (contributions.length) {
+        const pullRequestURL = await repository.createPullRequestFromFiles({
+            title: `docs: add ${who} as a contributor`,
+            body: `Adds @${who} as a contributor for ${contributions.join(
+                ', ',
+            )}.\n\nThis was requested by ${commentReply.replyingToWho()} [in this comment](${commentReply.replyingToWhere()})`,
+            filesByPath: filesByPathToUpdate,
+            branchName: `all-contributors/add-${who}`,
+        })
 
-    commentReply.reply(
-        `I've put up [a pull request](${pullRequestURL}) to add @${who}! :tada:`,
-    )
+        commentReply.reply(
+            `I've put up [a pull request](${pullRequestURL}) to add @${who}! :tada:`,
+        )
+    } else {
+        commentReply.reply(
+            `I could not find any contributions associated to @${who}! :sod:`,
```

Comment:
Like the use of emojis.

I think what we're after here is `i couldn't determine any contributions to add, did you specify contributions?`

Reply:
I've changed the comment to your suggestion.